internal/network/command_list.go

```diff
@@ -37,6 +38,21 @@ func (c *command) list(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	// Sort networks by DisplayName, then Cloud, then Region, then CreatedAt ASC.
+	sort.Slice(networks, func(i, j int) bool {
+		if networks[i].Spec.GetDisplayName() != networks[j].Spec.GetDisplayName() {
```

Comment:
Now that I think about it again, let's not sort by display name at all. it wouldn't be very useful, especially when the display name is optional

Sort the list of networks by
* cloud
* Region
* created timestamp

cc: @kabraham13